create_jenkins_job.py

```diff
@@ -77,7 +77,7 @@ def main(argv=None):
         'use_osrf_connext_debs_default': 'false',
         'use_fastrtps_default': 'true',
         'use_opensplice_default': 'false',
-        'ament_build_args_default': '--parallel --cmake-args -DSECURITY=ON --',
+        'ament_build_args_default': '--parallel --cmake-args -DSECURITY=ON',
```

Comment:
Oh, right, I see.  I misunderstood the intent of that.

This actually points out a weakness in this approach, which I don't quite know how to resolve yet.  If the user has a parameter like `--cmake-args -DSECURITY=ON --`, then the command-line ends up resolving to something like:

`--cmake-args -DSECURITY=ON -- --- --ament-test-args`

Which causes the run_ros2_batch.py command-line parser to explode.  We could fix that in the parser, but it doesn't quite seem to be the correct thing to do; that -- --- is gibberish, and probably shouldn't be allowed.  I'm not sure how to handle it yet.